server/clip_server/torch-flow.yml

```diff
@@ -2,10 +2,14 @@ jtype: Flow
 version: '1'
 with:
   port: 51000
+  monitoring: true
+  port_monitoring: 8000
 executors:
   - name: clip_t
     uses:
       jtype: CLIPEncoder
       metas:
         py_modules:
           - executors/clip_torch.py
+    monitoring: true
+    port_monitoring: 9000
```

Comment:
The issue is that there is always a default value pass to monitoring_port, so we do need another parameter to check to say if monitoring is enable or no. What I could do here is just use the default monitoring port so that the yaml does not contain redundant information

Reply:
The default value can be `None`, am I right?